### DISCUSSION TITLE
Failed to Authenticate using Tesla Proxy

### DISCUSSION THREAD

**Discussion by Boogmeister** [alandtse/tesla#950] [General]
Need some help.

After a lot of messing around finally got the proxy working as expected etc.

Now when trying to add the actual integration it doesn't work.

Says success and then the device is immediately "failed authentication.

Can't find anything odd in any of the logs.

Anyone have any ideas?

Tesla HTTP Proxy
Current version: 2.2.3

HAOS - 
Core
2024.4.4
Supervisor
2024.04.0
Operating System
12.2
Frontend
**20240404.2**

![Screenshot 2024-04-25 201258](https://github.com/alandtse/tesla/assets/12182528/e6d06e71-b8a9-4577-83fd-554d729ea4f2)
![Screenshot 2024-04-25 201329](https://github.com/alandtse/tesla/assets/12182528/df69e185-df1d-4540-abbf-1130968dc024)
![Screenshot 2024-04-25 201339](https://github.com/alandtse/tesla/assets/12182528/f793ef22-36ce-42bb-8cd8-99c4d35b7df5)
![Screenshot 2024-04-25 201352](https://github.com/alandtse/tesla/assets/12182528/9c136c84-aeab-4111-84b1-584608ce325a)
![Screenshot 2024-04-25 201657](https://github.com/alandtse/tesla/assets/12182528/fe07c4b0-72e8-4598-93d1-cfceadc7d55f)

**Comment by alandtse** [comment]
Issues with the Proxy probably should be raised [here](https://github.com/llamafilm/tesla-http-proxy-addon).
